app/src/main/res/values/colors.xml

```diff
@@ -16,4 +16,6 @@
     <color name="colorCoral">#f75f52</color>
     <color name="colorDarkGray">#404040</color>
     <color name="colorLightGray">#d0d0d0</color>
+
+    <color name="colorBlack">#131313</color>
```

Comment:
`colorBlack` might be a little misleading since this is not black; maybe you can find something like `colorBrownishBlack`? Or if you just use it for a specific element you can name the color specifically to that.